examples/recipes/plain_text_main.py

```diff
@@ -0,0 +1,21 @@
+import cgi
+import functools
+
+import falcon
+
+
+class TextHandler(falcon.media.BaseHandler):
+    DEFAULT_CHARSET = 'utf-8'
+
+    @classmethod
+    @functools.lru_cache
+    def _get_charset(cls, content_type):
+        _, params = cgi.parse_header(content_type)
```

Comment:
`cgi` has been removed from CPython 3.13+. You can use our own replacement: [`falcon.parse_header()`](https://falcon.readthedocs.io/en/stable/api/util.html#falcon.parse_header).